playwright_server

```diff
@@ -273,7 +274,7 @@ app.post('/command', async (req, res) => {
     } else if (command == 'on' && subject && spec[type].members.waitForEvent ) {
         try {
             var evt = args.shift();
-            const cb  = new Function (args.shift());
+            const cb  = new Function (...args);
```

Comment:
Interesting.  I'm not sure we'll ever need this, but it's a welcome addition.

Reply:
Indeed.  I realized that as I had to modify the xt test to survive dzil release.  Thanks for the great patchset.